### PR TITLE
Parse additional line names

### PR DESCRIPTION
**Pull request by juliuste**
Some DB lines contain an additional attribute called `addName` (only with the updated `body.ext`, for some reason), e.g.

```js
{
	name: 'RE 3177',
	number: '3177',
	icoX: 1,
	cls: 8,
	oprX: 0,
	prodCtx:
	{
		name: 'RE  3177',
		num: '3177',
		matchId: '1',
		catOut: 'RE',
		catOutS: 'RE',
		catOutL: 'Regional-Express',
		catIn: 'RE',
		catCode: '3',
		admin: '800154',
		addName: 'RE     1'
	},
	addName: 'RE 1'
}
```

This PR adds support for this (renamed to `line.additionalName`) in the DB profile.

### CODE REVIEW

**Comment by juliuste** [comment]
Only non-related tests are failing.

File: test/db.js [hunk 1]
```diff
@@ -366,6 +367,13 @@ test('station', co(function* (t) {
 	t.end()
 }))
 
+test('line with additionalName', co(function* (t) {
+	const departures = yield(client.departures(potsdamHbf, { duration: 12*60, products: { bus: false, suburban: false, tram: false } }))
```
Comment (derhuerst) [inline test/db.js]: Please remove the parenthesis after `yield` to make clear that `yield` is ECMAScript syntax, not a function.

Comment (juliuste) [reply test/db.js]: Fixed.

**Comment by juliuste** [comment]
LGTM

**Pull request merged**